run.sh

```diff
@@ -30,8 +30,7 @@ if echo "${OCP_AUTH_PROXY_MEMORY_LIMIT:-}" | grep -qE "^([[:digit:]]+)([GgMm])?i
         num="${DEFAULT_MIN}"
     fi
 
-    NODE_OPTIONS="--max-old-space-size=$((num / BYTES_PER_MEG))"
-    export NODE_OPTIONS
+    export NODE_OPTIONS="--max_old_space_size=$((num / ${BYTES_PER_MEG}))"
```

Comment:
Might I make a simple suggestion that will allow passing in node options from the dc:
```
export NODE_OPTIONS="${NODE_OPTIONS:---max_old_space_size=$((num / ${BYTES_PER_MEG}))}"
```
Then users can set NODE_OPTIONS in the env. in the dc to override this and/or provide additional options.